app/content/team/index.js

```diff
@@ -330,10 +342,12 @@ function Index() {
                           currentTeamId,
                           teamRef,
                         );
-                        invalidateMultipleKeys(queryClient, [["teamInfo"]]);
+                        await invalidateMultipleKeys(queryClient, [
```

Comment:
this might need try / catch / showdialog, see other comment about handleimageupload on this pr